src/containers/my-cooperations/cooperation-completion/CooperationCompletion.tsx

```diff
@@ -1,17 +1,38 @@
+import { useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import Divider from '@mui/material/Divider'
 
 import SettingItem from '~/components/setting-item/SettingItem'
 import AppButton from '~/components/app-button/AppButton'
+import AppSelect from '~/components/app-select/AppSelect'
 
+import { cooperationAccessValues } from '~/containers/my-cooperations/cooperation-completion/CooperationCompletion.constants'
 import { styles } from '~/containers/my-cooperations/cooperation-completion/CooperationCompletion.styles'
-import { ButtonVariantEnum, SizeEnum } from '~/types'
+import {
+  ButtonVariantEnum,
+  SizeEnum,
+  CooperationMaterialsAccessEnum,
+  UserRoleEnum
+} from '~/types'
 
-const CooperationCompletion = () => {
+interface CooperationCompletionProps {
+  onCloseCooperation: () => void
+  userRole: UserRoleEnum | ''
```

Comment:
```suggestion
  userRole: UserRoleEnum
```
Is any chance for userRole to be `''`?